### PR TITLE
added simple query implementation

### PR DESCRIPTION
**Pull request by alswang18**
new PR addressing issues in this previous PR: https://github.com/sfu-db/connector-x/pull/322

### CODE REVIEW

File: connectorx/src/get_arrow2.rs [hunk 1]
```diff
@@ -114,6 +115,36 @@ pub fn get_arrow2(
...
+                ("simple", Some(tls_conn)) => {
+                    let sb = PostgresSource::<SimpleProtocol, MakeTlsConnector>::new(
+                        config,
+                        tls_conn,
+                        queries.len(),
+                    )?;
+                    let dispatcher = Dispatcher::<
+                        _,
+                        _,
+                        PostgresArrowTransport<SimpleProtocol, MakeTlsConnector>,
```
Comment (wangxiaoying) [inline connectorx/src/get_arrow2.rs:127]: This transport should be `PostgresArrow2Transport`, the same applies to the below one.

Comment (alswang18) [reply connectorx/src/get_arrow2.rs]: fixed it

**Comment by wangxiaoying** [comment]
1. format code
2. need to add simple protocol support for pandas too at [here](https://github.com/sfu-db/connector-x/blob/main/connectorx-python/src/pandas/mod.rs)

**Review by wangxiaoying — CHANGES_REQUESTED** [review]
1. format code
2. need to add simple protocol support for pandas too at [here](https://github.com/sfu-db/connector-x/blob/main/connectorx-python/src/pandas/mod.rs)

**Comment by alswang18** [comment]
> 1. format code
> 2. need to add simple protocol support for pandas too at [here](https://github.com/sfu-db/connector-x/blob/main/connectorx-python/src/pandas/mod.rs)

just made the changes requested @wangxiaoying

**Pull request merged**